framework/src/base/MooseObject.C

```diff
@@ -20,15 +20,15 @@ template<>
 InputParameters validParams<MooseObject>()
 {
   InputParameters params;
+  params.addParam<std::string>("control_tag", "Adds a user-defined label for accessing object parameters via control logic.");
```

Comment:
Hmmm... once again brings up questions of whether we should have our own InputParameters object.

I'd prefer `params.addControlTag(std::string tag)` that adds the tag to the parameter.